### DISCUSSION TITLE
API Routes for serving Front End & authentication

### DISCUSSION THREAD

**Discussion by ezek-b** [vhessam/hessam-cms#54] [General]
Hi All,

First, thank you for sharing this incredible package,
I would like to evaluate your package for my use case. I am building a project where the back is managed by multiple modules, one is built with Laravel 8.

1. The front will sit in a dedicated repo and will be built in REACT. I would like to know if it will be possible to have an API route for displaying the blog articles in the Front End. All the admin part will remain as existing.
2. Regarding authentication, what is the basic configuration for the blog admins ? Is there a login and password with a dedicated auth flow?

Thanks
Ezekiel

**Comment by samberrry** [comment]
@ezek-b 
Hi, 
thanks for your interest. We have some plans for REST API as it's a good idea to provide headless CMS for this simple package. Really interesting. 
**Binshops** is now official partner with Vue Storefront, then we will connect it to this great headless approach ;) 

Thanks
